cmd/flowrunner/runner_test.go

```diff
@@ -168,27 +170,53 @@ func newTestHTTPServer() *httptest.Server {
 	return server
 }
 
+type fixedUUIDGenerator struct {
+	uuids []string
+	index int
+}
+
+func fixedUUIDGeneratorFromFile(path string) (*fixedUUIDGenerator, error) {
```

Comment:
Does this actually need to be from a file? Can't we seed the random number generator and get deterministic UUIDs?